package.json

```diff
@@ -12,7 +12,9 @@
     "test": "cross-env BABEL_ENV=cjs jest --coverage",
     "test:watch": "npm test -- --watch",
     "release": "node scripts/release.js",
-    "postinstall": "node scripts/installNestedPackageDeps.js"
+    "postinstall": "node scripts/installNestedPackageDeps.js",
+    "format": "rm -rf src/packages/*/node_modules;prettier --semi false --trailing-comma es5 --single-quote --write '{src/,scripts}/**/*.js'",
```

Comment:
Fixed, can't get why it works but it works, I thought multiple path templates are combined using OR operation but here is something different. May you say where I can read about?